src/Configuration/DefaultConfiguration.php

```diff
@@ -407,7 +407,7 @@ private function guessSymfonyDirectoryStructure(int $symfonyMajorVersion, $symfo
         // TODO: Be a bit more clever and for example take composer.json extra configuration into account
         if (2 === $symfonyMajorVersion) {
             $this->_symfonyDirectoryStructureVersion = self::SYMFONY_2;
-        } elseif (3 === $symfonyMajorVersion && 4 < $symfonyMinorVersion) {
+        } elseif (3 === $symfonyMajorVersion && 3 < $symfonyMinorVersion) {
```

Comment:
I think I botched the comparison because of the order of the operands. This applies to Symfony versions of "> 3.3" and the check below in line 412 only applies to "<= 3.4". I need to change this completely.